testsuite/src/it/java/io/quarkus/ts/startstop/utils/MvnCmds.java

```diff
@@ -25,11 +26,12 @@
     GENERATOR(new String[][]{
             new String[]{
                     "mvn",
-                    "io.quarkus.platform:quarkus-maven-plugin:" + getQuarkusVersion() + ":create",
+                    getQuarkusGroupId() + ":quarkus-maven-plugin:" + getQuarkusVersion() + ":create",
```

Comment:
So we are using productized maven plugin?

Reply:
Note from discussion around quarkus maven plugin:

yes, we promote io.quarkus.platform:quarkus-maven-plugin in the community and com.redhat.quarkus.platform:quarkus-maven-plugin in the product io.quarkus:quarkus-maven-plugin isn't supposed to be used